source/core/Console.cpp

```diff
@@ -602,7 +602,7 @@ namespace GTA
 	{
 		array<Script ^> ^scripts = ScriptDomain::CurrentDomain->RunningScripts;
 
-		if (scripts->Length == 0)
+		if (scripts->Length <= 1)
```

Comment:
@crosire Natives would be fixed by including natives.h in the console file. Key management could be taken care of in the void ManagedKeyboardMessage function. the only issue rn i cant think of would be how to get custom script commands working, but that should be solveable by executing console commands in the current script domain